### ISSUE TITLE
Provide Cluster Context in logs

### ISSUE THREAD

**Issue by benarent** [gravitational/teleport#3751]
### Feature Request
When aggregating and centralising Teleport logs, it's helpful to know which cluster the event happened on, without having to build extra tooling.  An example `session.start` event. 

**Example from `sudo tail -f /var/lib/teleport/log/events.log`**
```json
{"addr.local":"66.220.18.10:3022","addr.remote":"73.202.138.51:46798","code":"T2000I","ei":0,"event":"session.start","login":"root","namespace":"default","server_hostname":"teleport5auth","server_id":"a54859b4-0eba-4f36-af6a-452cb8379e38","server_labels":{"db_role":"master","db_type":"postgres"},"sid":"85eeb0a1-9aec-11ea-a0bd-125400432332","size":"80:25","time":"2020-05-20T22:52:04.111Z","uid":"b8bd5094-cb9f-4520-8f94-b71865fd9e19","user":"benarent"}
```
Formatted for easier analysis 
```json
{
   "addr.local":"66.220.18.10:3022",
   "addr.remote":"73.202.138.51:46798",
   "code":"T2000I",
   "ei":0,
   "event":"session.start",
   "login":"root",
   "namespace":"default",
   "server_hostname":"teleport5auth",
   "server_id":"a54859b4-0eba-4f36-af6a-452cb8379e38",
   "server_labels":{
      "db_role":"master",
      "db_type":"postgres"
   },
   "sid":"85eeb0a1-9aec-11ea-a0bd-125400432332",
   "size":"80:25",
   "time":"2020-05-20T22:52:04.111Z",
   "uid":"b8bd5094-cb9f-4520-8f94-b71865fd9e19",
   "user":"benarent"
}
```

sid = Session ID, can we used to retrieve session playback later, or share session. 

We should either provide `cluster_name` or proposed Alias https://github.com/gravitational/teleport/issues/3604. Since  `cluster_name` is immutable and baked in the SSH CA I would suggest adding that.

```diff
{
   "addr.local":"66.220.18.10:3022",
   "addr.remote":"73.202.138.51:46798",
   "code":"T2000I",
   "ei":0,
   "event":"session.start",
   "login":"root",
   "namespace":"default",
   "server_hostname":"teleport5auth",
   "server_id":"a54859b4-0eba-4f36-af6a-452cb8379e38",
   "server_labels":{
      "db_role":"master",
      "db_type":"postgres"
   },
   "sid":"85eeb0a1-9aec-11ea-a0bd-125400432332",
   "size":"80:25",
   "time":"2020-05-20T22:52:04.111Z",
   "uid":"b8bd5094-cb9f-4520-8f94-b71865fd9e19",
   "user":"benarent"
+   "cluster_name":"teleport-431"
}
```

### Motivation
It makes logs useful when aggregated across many clusters. 

### Who's it for?
OSS User, Pro, Enterprise

**Comment by travelton** [comment]
Note: This seems to have been merged into 6.0. Though, this ticket was not part of that milestone. Notified customers waiting on this.

**Comment by benarent** [comment]
@travelton I was just testing this and most of it made it's way but I found a couple of bugs. https://github.com/gravitational/teleport/issues/5856
